source/state/vault/index.ts

```diff
@@ -308,13 +308,23 @@ const VaultState = createSlice({
     },
     setFaucetModalState: (
       state: IVaultState,
-      action: PayloadAction<number>
+      action: PayloadAction<{ chainId: number; isFirstTime?: boolean }>
     ) => {
+      const { chainId, isFirstTime } = action.payload;
       if (state.isBitcoinBased) {
         return;
       }
 
-      state.faucetModal[action.payload] = false;
+      if (isFirstTime && !state.isBitcoinBased) {
```

Comment:
this will be replaced with the migration, right?

Reply:
probably yes, we can discuss whether it would still be interesting to leave it to have an extra "protection"